shared/src/main/java/com/sample/shared/RetryProperties.java

```diff
@@ -0,0 +1,105 @@
+package com.sample.shared;
+
+import com.google.api.gax.retrying.RetrySettings;
+import org.threeten.bp.Duration;
+
+/** Retry settings. */
+public class RetryProperties {
```

Comment:
Maybe instead of a shared object, `RetryProperties` will need to be generated per-module with fields consistent with client library logic in [`RetrySettingsComposer`](https://github.com/googleapis/gapic-generator-java/blob/17ccc21e395a0e3f0a81724e069ae5e0b1e02b6b/src/main/java/com/google/api/generator/gapic/composer/common/RetrySettingsComposer.java#L495-L588)